patches/chromium/add_realloc.patch

```diff
@@ -1,83 +0,0 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Shelley Vohr <shelley.vohr@gmail.com>
-Date: Thu, 20 Sep 2018 17:44:26 -0700
-Subject: add_realloc.patch
-
-Blink overrides ArrayBuffer's allocator with its own one, while Node simply
-uses malloc and free, so we need to use v8's allocator in Node. As part of the
-10.6.0 upgrade, we needed to make SerializerDelegate accept an allocator
-argument in its constructor, and override ReallocateBufferMemory and
-FreeBufferMemory to use the allocator. We cannot simply allocate and then memcpy
-when we override ReallocateBufferMemory, so we therefore need to implement
-Realloc on the v8 side and correspondingly in gin.
-
-diff --git a/gin/array_buffer.cc b/gin/array_buffer.cc
-index 210760801f1d027196111631d34bab3eb5a10792..cdfdf91841b5f2feb248b0c5890ddcfdb5a8f9ce 100644
---- a/gin/array_buffer.cc
-+++ b/gin/array_buffer.cc
-@@ -37,6 +37,10 @@ void* ArrayBufferAllocator::AllocateUninitialized(size_t length) {
-   return malloc(length);
- }
- 
-+void* ArrayBufferAllocator::Realloc(void* data, size_t length) {
-+  return realloc(data, length);
-+}
-+
- void ArrayBufferAllocator::Free(void* data, size_t length) {
-   free(data);
- }
-diff --git a/gin/array_buffer.h b/gin/array_buffer.h
-index aef43319737398848dc40a3ab2d9e959ebb399f6..3c6dbc1ec10666b35d68e107a7a694c59c78a59f 100644
---- a/gin/array_buffer.h
-+++ b/gin/array_buffer.h
-@@ -20,6 +20,7 @@ class ArrayBufferAllocator : public v8::ArrayBuffer::Allocator {
-  public:
-   void* Allocate(size_t length) override;
-   void* AllocateUninitialized(size_t length) override;
-+  void* Realloc(void* data, size_t length) override;
-   void Free(void* data, size_t length) override;
- 
-   GIN_EXPORT static ArrayBufferAllocator* SharedInstance();
-diff --git a/third_party/blink/renderer/bindings/core/v8/v8_initializer.cc b/third_party/blink/renderer/bindings/core/v8/v8_initializer.cc
-index 54c0c689bcf1f68ebc11fc76f73d36c2439c7a0d..04fc1864937150f19945ec06a28f0601be286d0d 100644
---- a/third_party/blink/renderer/bindings/core/v8/v8_initializer.cc
-+++ b/third_party/blink/renderer/bindings/core/v8/v8_initializer.cc
-@@ -698,6 +698,10 @@ class ArrayBufferAllocator : public v8::ArrayBuffer::Allocator {
-     return result;
-   }
- 
-+  void* Realloc(void* data, size_t size) override {
-+    return ArrayBufferContents::Realloc(data, size);
```

Comment:
Doesn't the blink patch need to survive because of https://github.com/electron/electron/blob/master/patches/v8/add_realloc.patch ?

Reply:
hm good point, let me remove that one also and see if things still work

Reply:
@deepak1556 looks like neither of these patches is needed any longer 🎉